File: ios/C3POReactNativeExpoNotif.m
#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>

@interface RCT_EXTERN_MODULE(C3POReactNativeExpoNotif, RCTEventEmitter)

    + (BOOL)requiresMainQueueSetup
    {
        return YES;
    }
    
    RCT_EXTERN_METHOD(
        supportedEvents:
    )

@end
    

File: ios/C3POReactNativeExpoNotif.swift
//
//  C3POReactNativeExpoNotif.swift
//  C3POReactNativeExpoNotif
//
//  Created by Florian Guitton on 15/04/2020.
//

import Foundation

@objc(C3POReactNativeExpoNotif)
open class C3POReactNativeExpoNotif: RCTEventEmitter {

    public override static func moduleName() -> String! {
        return "C3POReactNativeExpoNotif"
    }

    @objc
    open override func supportedEvents() -> [String] {
        return ["onDeviceFound", "onBluetoothStateChange"]
    }
}
